vignettes/collaboration.Rmd

```diff
@@ -0,0 +1,261 @@
+    padding: 0 5px;
+    color: hotpink;
+}
+.bob {
+    /* background-color: lightblue; */
+    /* border: 3px solid blue; */
+    /* font-weight: bold; */
+    padding-right: 30px
+}
+div pre.bob:after {
+    content:"Bob";
+    position: absolute;
+    right: 0px;
+    top: 0px;
+    font-size: 80%;
+    font-weight: normal;
+    font-family: sans-serif;
+    padding: 0 5px;
+    color: steelblue4;
+}
+```
+
+One of the core purposes of `orderly2` is to allow collaborative workflows. For this to work we need to be able to find packets that were run by someone else, somewhere else, or to share packets that we have run with others. To do that, `orderly2` has the idea of "locations".
+
+Locations behave in a similar way to "remotes" in `git`, and some of the terminology for interacting with them is similar (notably we can `pull` from and perhaps `push` to a location). However, we have used a different name because every `orderly2` working directory has at least one location - the `local` location - which is not remote at all. We also imagine that almost everyone using `orderly2` will be using `git` and as `git` remotes and `orderly2` locations refer to totally different things this slightly different terminology may help keep things clearer.
+
+Packets that are distributed by locations might have been created by `orderly2`, but may have been created by some other system that implements the `outpack` spec (see `vignette("metadata")`).  As a result, through this documentation you will see reference to software or details to do with `outpack` but if you only use `orderly2` then you can read that as being synonymous with `orderly2`.
+
+# Type of location
+
+`orderly2` supports two types of locations by default:
+
+* `path`: these are any `orderly2` working copy that you can find on your filesystem. Note that this could be a copy that other people can see (for example on a network share, or on a cloud-synced file system such as Dropbox or OneDrive)
+* `http`: these require running an HTTP API, either via [`outpack_server`](https://github.com/mrc-ide/outpack_server) or [`packit`](https://github.com/mrc-ide/packit)
+
+The location system is somewhat extensible, but the details of this are subject to change.  There is [`outpack.sharepoint`](https://github.com/mrc-ide/outpack.sharepoint) for using SharePoint, and we may write other backends later.
+
+All the documentation below will behave the same regardless of where the location is stored and the mechanism of transport, so we can focus instead on workflows.
+
+# An example
+
+Suppose we have two researchers, [Alice and Bob](https://en.wikipedia.org/wiki/Alice_and_Bob), who are collaborating on a piece of analysis. For the purposes of this example (and we find the most common situation in practice) they share a source tree with their analysis in git.
+<!-- The source for the repository that they are using is on GitHub as
+[`reside-ic/orderly-example`](https://github.com/reside-ic/orderly-example).
```

Comment:
You don't want to make this visible to the reader? I could see it being useful to be able to read the examples

Reply:
this does not exist yet!